index.html

```diff
@@ -30,16 +44,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" style="background: teal" >Box 1</div>
```

Comment:
While inline styling gets the job done for this sprint, it isn't best practice becaue it fragments your code and makes it harder to edit in the future. you can always use a CSS Selector (e.i. classes, ids, nth-child()....etc) to target each box independantly and apply custom styling